src/background/service/preference.ts

```diff
@@ -23,6 +31,7 @@ interface PreferenceStore {
   isDefaultWallet: boolean;
   lastTimeSendToken: Record<string, TokenItem>;
   walletSavedList: [];
+  gasCache: ChainGas[];
```

Comment:
looks like gasCache should be a Map instead Array?

Reply:
done